tests/test_dense.py

```diff
@@ -1,101 +1,129 @@
-import unittest
 import sys
-from mock_data.dense_matrix_19_genes_100k_cells_txt.gene_models_0 import gene_models
+import unittest
+from unittest.mock import Mock, patch
 
 sys.path.append("../ingest")
-from dense import Dense
+from expression_files.dense_ingestor import DenseIngestor
+from mock_data.dense_matrix_19_genes_100k_cells_txt.gene_models_0 import \
+    gene_models
 
 
 class TestDense(unittest.TestCase):
-    def test_validate_gene_keyword_false(self):
+
```

Comment:
Looking good.  I think we need to add tests that we handle single/double quotes in the header row (are there any other special characters we should strip from the headers?), and detect duplicates correctly even with quotes.

@jlchang  can you take a look at this file and let us know what are other expression file edge parsing cases we need to handle?

Reply:
This is handled is this [test case](https://github.com/broadinstitute/scp-ingest-pipeline/blob/3e6e92b888a063579686fe376e29c1445b6ffefd/tests/test_dense.py#L35).

Reply:
eb08461